pandas/core/groupby/groupby.py

```diff
@@ -1167,8 +1167,8 @@ def _wrap_applied_output(
     @final
     def _numba_prep(self, data: DataFrame):
         ids, _, ngroups = self.grouper.group_info
-        sorted_index = get_group_index_sorter(ids, ngroups)
-        sorted_ids = algorithms.take_nd(ids, sorted_index, allow_fill=False)
+        sorted_index = self.grouper._sort_idx
+        sorted_ids = self.grouper._sorted_ids
```

Comment:
Would it be more consistent to call it `_sorted_idx`? Or is there a reason for `sort` over `sorted`?

Reply:
i just kept the existing name.  best guess is that this is the object we use _to_ sort, not the sorted object itself.